src/audio_worklet.js

```diff
@@ -219,5 +189,11 @@ class BootstrapMessages extends AudioWorkletProcessor {
   }
 };
 
+
 // Register the dummy processor that will just receive messages.
+#if WEBAUDIO_DEBUG
+console.log('new audio worklet, calling registerProcessor');
+#endif
 registerProcessor('message', BootstrapMessages);
```

Comment:
Something I changed in my earlier attempt to inline aw.js - let's rename it to something like `em-boostrap`. This is a globally registered name, and may conflict with user-defined ones if any.

Reply:
Lets do that kind of orthogonal improvement as a followup.

Reply:
I don't think it's really orthogonal, because now the main JS itself is used as audio worklet instead of having a separate file, so the naming conflicts if any become more likely.

Reply:
Basically: before this if you loaded Emscripten-generated JS in your own custom audio worklet, it's fine, no conflict would occur because the audio worklet itself lives in a separate file anyway, but now if you load it, the `registerProcessor` function will be invoked and it might conflict name-wise, so it's more important to keep it under unique prefix.

Name like `message` is just way too generic.

Reply:
Ideally we the generated code would know that its running in an AW (like we do with wasm workers and pthreads via the `em-ww` and `em-pthread` globals).. but you are right, the way AW seems to be started there is not way to inject any kind of options/settings.    It seems rather odd that nothing like is available when calling `addModule` (https://developer.mozilla.org/en-US/docs/Web/API/Worklet/addModule#options).  For `new Worker` we use the name field: https://developer.mozilla.org/en-US/docs/Web/API/Worker/Worker#name.  Maybe we can ask them to add that.

In the mean time I will make this change :)